manifests/u/UnigramDev/Unigram/8.9.7682.0/UnigramDev.Unigram.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: UnigramDev.Unigram
+PackageVersion: 8.9.7682.0
+Installers:
+- Platform:
+  - Windows.Universal
+  MinimumOSVersion: 10.0.18362.0
+  Architecture: x64
+  InstallerType: msix
+  InstallerUrl: https://github.com/UnigramDev/Unigram/releases/download/latest/Unigram.msixbundle

```

Comment:
Please use a versioned URL and not a URL to Latest

Reply:
they update a github release without creating a new one(https://github.com/UnigramDev/Unigram/releases/tag/latest)